packages/yoastseo/spec/languageProcessing/languages/ja/helpers/countCharactersSpec.js

```diff
@@ -0,0 +1,12 @@
+import countCharactersFunction from "../../../../../src/languageProcessing/languages/ja/helpers/countCharacters.js";
+
+describe( "counts characters in a string", function() {
+	it( "returns the number of characters", function() {
+		expect( countCharactersFunction( "this is a string" ) ).toBe( 16 );
+		expect( countCharactersFunction( "Низът в компютърните науки е крайна поредица от символи " +
```

Comment:
Is there a reason in using Russian text for the unit test in this Japanese specific helper file? It is not problematic, but I think in general it's better to only have unit tests that are relevant for the Japanese helper that is being introduced here.

Reply:
Yeah my thought process was that this helper is not in nature Japanese-specific, it simply counts every single character, and should be able to do it for any and every language. This is why I wanted to try it with different languages: English, Bulgarian, and Japanese. 
I will keep in mind to include texts only in that particular language for language-specific specs.

Reply:
Then it's okay to keep the unit tests there. But then adding an appropriate comment on the unit test is necessary. 
So something like:
````
it( "makes sure the count character function also works for other characters other than Japanese", function() {
		expect( countCharactersFunction( "Низът в компютърните науки е крайна поредица от символи " +
"(представляващи краен брой знаци)." ) ).toBe( 90 );
	} );
````